content/03.methods.md

```diff
@@ -66,6 +69,17 @@ That is, each training set/test set combination had equal proportions of each ca
 To choose the elastic net hyperparameters, we used 3-fold nested cross-validation, with a grid search over the same hyperparameter ranges used in Way et al. 2020 [@doi:10.1186/s13059-020-02021-3]: $\lambda$ = [0.15, 0.16, 0.2, 0.25, 0.3, 0.4] and $\alpha$ = [0.1, 0.13, 0.15, 0.2, 0.25, 0.3].
 Using the grid search results, for each evaluation fold we selected the set of hyperparameters with the optimal area under the receiver-operator curve (AUROC), averaged over the three inner folds.
 
+Although we stratified our train and test datasets for cancer type and sample type, we opted not to stratify by label.
+In other words, we did not ensure that cross-validation splits had proportions of positive (mutated) and negative (non-mutated) samples that were exactly equal to each other; we instead sampled randomly with respect to labels.
+Label-stratified cross-validation chooses test sets that have exactly the same label balance as the training set, which could lead to an overly optimistic model evaluation, or could choose a model that is less robust to shifts in the label proportion than one that is trained and evaluated without controlling label balance.
```

Comment:
It seems like it might be overly optimistic or even overly pessimistic. I wonder if the key is that there is also structure between the labels (i.e., mutual exclusivity or similar).